dotCMS/src/main/resources/dotmarketing-config.properties

```diff
@@ -522,6 +522,14 @@ CACHE_INVALIDATION_TRANSPORT_CLASS=com.dotcms.cache.transport.HazelcastCacheTran
 ## Cache elasticsearch query results
 ES_CACHE_SEARCH_QUERIES=true
 
+# This Property can be set to a positive numeric value or a boolean value
+# if set to false ES will stop retrieving the hits Count. DO NOT SET IT TO FALSE.
+# IF set to true  ES will track all hits beyond the 10K items limit. This might imply an impact performance wise.
+# By default we're providing a max number.
+# If commented out ES will default to 10K again.
+# https://www.elastic.co/guide/en/elasticsearch/reference/7.10/search-your-data.html#track-total-hits
+ES_TRACK_TOTAL_HITS=1000000
```

Comment:
Our default should be 1,000,000 and this should not be set by default - we should not need to add this property to the `dotmarketing-config.properties` file.